test/Functions/OrderEntry/OrderEntryTransactionLineCreateTest.ts

```diff
@@ -114,7 +116,7 @@ describe("OrderEntryTransactionLineCreate", () => {
         record.revRecStartDate = new Date(2015, 5, 30);
         record.revRecEndDate = new Date(2015, 6, 31);
         record.renewalMacro = "Quarterly";
-        record.fulfillmentStatus = "Complete";
+        record.fulfillmentStatus = "Undelivered";
```

Comment:
is the part of the functional requirement change to use `Undelivered` now instead of `Complete` like before?

Reply:
Correct. 
https://developer.intacct.com/api/order-entry/order-entry-transactions/#create-order-entry-transaction-legacy